supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -266,7 +266,7 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
   private def modifyCommandForDocker(jobCmd: Try[String], localizedInputs: CallInputs): Try[String] = {
     Try {
       val dockerInputDataVol = localizedInputs.collect {
-        case (k, v) => v match {
+        case (k, v) if v.wdlType == WdlFileType || v.wdlType == WdlArrayType(WdlFileType) => v match {
```

Comment:
It's kind of strange the type is tested on two consecutive lines.  You might have the PF something like:

```scala
      val dockerInputDataVol = localizedInputs.collect {
        case (k, v) if v.wdlType == WdlFileType =>
          val limit = v.valueString.lastIndexOf("/")
          Seq(v.valueString.substring(0, limit))
        case (k, v) if v.wdlType == WdlArrayType(WdlFileType) =>
          v.asInstanceOf[WdlArray].value map { file =>
            val limit = v.valueString.lastIndexOf("/")
            v.valueString.substring(0, limit)
          }
      }.flatten.toSeq
```